README.md

```diff
@@ -65,4 +65,39 @@ The following command will run a **predeploy** script to make the conversion and
 
 ```sh
 $ npm run deploy
+```
+
+# Deploying Firebase Storage Rules!
+First we need to create **target names** using firebase CLI as follows:
+
+```sh
+$ firebase target:apply storage target-name resource-name
+```
+
+where:
+1. **target-name**: is a unique identifier.
+2. **resource-name**: is the name of the bucet on Firebase storage.
+ 
+Then in the **firebase.json** add a new "storage" key with an array of target-names and their associated rule files like the following example:
+
+```json
+{
+  "storage": [{
+    "target": "se_uploads",
+    "rules": "./functions/storage.sound-editing.uploads.rules"
+  }]
+}
+```
+
+Then to deploy the created storage rules, run the following:
+
+```sh
+$ firebase deploy --only storage:target-name
+```
```

Comment:
Above just keep the commands that a developer needs to actually issue. Don't copy the theory from the documentation, better to give a link. No need describing what to add to `firebase.json` as soon as you already added this and committed. And the target name is already defined, so just describe what is to be done on the developer computer.